.ko.yaml

```diff
@@ -45,3 +45,199 @@ baseImageOverrides:
   k8s.io/test-infra/gcsweb/cmd/gcsweb: gcr.io/k8s-prow/alpine:v20200713-e9b3d9d
   # prow integration test
   k8s.io/test-infra/prow/test/integration/fakeghserver: gcr.io/k8s-prow/alpine:v20200713-e9b3d9d
+
+builds:
```

Comment:
I think we need also cherrypicker, refresh, needs-rebase, maybe also `label_sync` and `commenter`. Is there a reason why we shouldn't do this for all images in the list above?

Reply:
added

Reply:
`needs-rebase` is still missing.

Reply:
added